diff --git a/src/main/index.ts b/src/main/index.ts
index 9e9244f..ccc2374 100644
--- a/src/main/index.ts
+++ b/src/main/index.ts
@@ -43,7 +43,19 @@ function init() {
             "AcceleratedVideoDecodeLinuxGL",
             "AcceleratedVideoEncoder",
             "AcceleratedVideoDecoder",
-            "AcceleratedVideoDecodeLinuxZeroCopyGL"
+            "AcceleratedVideoDecodeLinuxZeroCopyGL",
+            "VaapiVideoDecodeLinuxGL",
+            "VaapiVideoEncoder",
+            "VaapiVideoDecoder",
+            "VulkanFromANGLE",
+            "DefaultANGLEVulkan",
+            "VaapiIgnoreDriverChecks",
+            "VaapiVP8Encoder",
+            "VaapiVP9Encoder",
+            "VaapiAV1Encoder",
+            "WaylandWindowDecorations",
+            "UseOzonePlatform",
+            "WebRTCPipeWireCapturer"
         );
     }
 
@@ -51,6 +63,27 @@ function init() {
         app.commandLine.appendSwitch("disable-smooth-scrolling");
     }
 
+    app.commandLine.appendSwitch("disable-renderer-backgrounding");
+    app.commandLine.appendSwitch("disable-background-timer-throttling");
+    app.commandLine.appendSwitch("disable-backgrounding-occluded-windows");
+    app.commandLine.appendSwitch("enable-zero-copy");
+    app.commandLine.appendSwitch("use-gl=angle");
+    app.commandLine.appendSwitch("use-angle=gl");
+    app.commandLine.appendSwitch("use-vulkan");
+    app.commandLine.appendSwitch("enable-oop-rasterization");
+    app.commandLine.appendSwitch("enable-raw-draw");
+    app.commandLine.appendSwitch("enable-gpu-rasterization");
+    app.commandLine.appendSwitch("enable-gpu-compositing");
+    app.commandLine.appendSwitch("enable-native-gpu-memory-buffers");
+    app.commandLine.appendSwitch("enable-accelerated-2d-canvas");
+    app.commandLine.appendSwitch("enable-accelerated-video-decode");
+    app.commandLine.appendSwitch("enable-accelerated-mjpeg-decode");
+    app.commandLine.appendSwitch("disable-gpu-vsync");
+    app.commandLine.appendSwitch("disable-frame-rate-limit");
+    app.commandLine.appendSwitch("ozone-platform-hint=auto");
+    app.commandLine.appendSwitch("enable-webrtc-pipewire-capturer");
+    app.commandLine.appendSwitch("ozone-platform=wayland");  
+
     // disable renderer backgrounding to prevent the app from unloading when in the background
     // https://github.com/electron/electron/issues/2822
     // https://github.com/GoogleChrome/chrome-launcher/blob/5a27dd574d47a75fec0fb50f7b774ebf8a9791ba/docs/chrome-flags-for-tools.md#task-throttling
diff --git a/src/renderer/components/ScreenSharePicker.tsx b/src/renderer/components/ScreenSharePicker.tsx
index f69f9d5..360348f 100644
--- a/src/renderer/components/ScreenSharePicker.tsx
+++ b/src/renderer/components/ScreenSharePicker.tsx
@@ -85,9 +85,9 @@ addPatch({
         const width = Math.round(height * (16 / 9));
 
         Object.assign(opts, {
-            bitrateMin: 500000,
-            bitrateMax: 8000000,
-            bitrateTarget: 600000
+            bitrateMin: 10000000,
+            bitrateMax: 60000000,
+            bitrateTarget: 32000000 
         });
         if (opts?.encode) {
             Object.assign(opts.encode, {
