diff --git a/src/launch/command.cpp b/src/launch/command.cpp
index 94fe239..1b24f9b 100644
--- a/src/launch/command.cpp
+++ b/src/launch/command.cpp
@@ -84,7 +84,8 @@ int locateConfigFile(CommandState& cmd, QString& path) {
 	} else {
 		auto manifestPath = *cmd.config.manifest;
 		if (manifestPath.isEmpty()) {
-			auto configDir = QDir(QStandardPaths::writableLocation(QStandardPaths::AppConfigLocation));
+			auto configDir =
+			    QDir(QStandardPaths::writableLocation(QStandardPaths::ConfigLocation) + "/quickshell");
 			auto path = configDir.filePath("manifest.conf");
 			if (QFileInfo(path).isFile()) manifestPath = path;
 		}
diff --git a/src/launch/main.cpp b/src/launch/main.cpp
index 7a801fc..1656e40 100644
--- a/src/launch/main.cpp
+++ b/src/launch/main.cpp
@@ -103,6 +103,7 @@ void exitDaemon(int code) {
 
 int main(int argc, char** argv) {
 	QCoreApplication::setApplicationName("quickshell");
+	QCoreApplication::setOrganizationName("quickshell");
 
 #if CRASH_REPORTER
 	qsCheckCrash(argc, argv);
